app/code/community/SomethingDigital/Modal/Block/Modal.php

```diff
@@ -12,6 +12,9 @@ public function getConfig($value)
     public function getModalContent() 
     {
         $blockId = $this->getConfig('block');
+        if(!$this->getLayout()) {
+            return;
```

Comment:
This would return null, if I'm not mistaken. Not sure if that would be rendered. It doesn't seem like there is a check that prevents the modal from rendering if there is no content. Might be better to return either return an empty string or maybe better, an error message so that it's not a silent failure. What are your thoughts on error handling for this?